libs/ui-lib/lib/common/components/ui/formik/CodeField.tsx

```diff
@@ -81,13 +105,16 @@ const CodeField = ({
             language={language}
             onEditorDidMount={(editor) => setMonacoEditor(editor)}
             downloadFileName={downloadFileName}
+            onCodeChange={(value: string) => {
+              onCodeChange(value);
+            }}
           />
         </FormGroup>
       </StackItem>
       <StackItem>
-        {errorMessage && (
+        {!isValid && (
           <HelperText fieldId={fieldId} isError>
-            {errorMessage}
+            {!isValidValue ? FILE_TYPE_MESSAGE : errorMessage}
```

Comment:
I'm not sure it's a good idea to hardcode error messages in a component that is supposed to be generic and reusable. I would prefer to see the validation logic on form level.